zerver/migrations/0001_initial.py

```diff
@@ -754,91 +638,103 @@ class Migration(migrations.Migration):
-                    django.core.validators.RegexValidator(
-                        message="Invalid characters in emoji name",
-                        regex="^[0-9a-zA-Z.\\-_]+(?<![.\\-_])$",
-                    ),
-                ]
+            index=models.Index(
+                django.db.models.functions.text.Upper("email"), name="upper_userprofile_email_idx"
             ),
         ),
-        migrations.AddField(
+        migrations.AddIndex(
             model_name="preregistrationuser",
-            name="realm_creation",
-            field=models.BooleanField(default=False),
-        ),
-        migrations.AddField(
-            model_name="attachment",
-            name="realm",
-            field=models.ForeignKey(
-                blank=True,
-                null=True,
-                on_delete=django.db.models.deletion.CASCADE,
-                to="zerver.Realm",
+            index=models.Index(
+                django.db.models.functions.text.Upper("email"),
+                name="upper_preregistration_email_idx",
             ),
         ),
-        migrations.AddField(
-            model_name="subscription",
-            name="pin_to_top",
-            field=models.BooleanField(default=False),
-        ),
-        migrations.AddField(
-            model_name="userprofile",
-            name="default_language",
-            field=models.CharField(default="en", max_length=50),
-        ),
-        migrations.AddField(
-            model_name="realm",
-            name="allow_message_editing",
-            field=models.BooleanField(default=True),
+        migrations.AddIndex(
+            model_name="message",
+            index=django.contrib.postgres.indexes.GinIndex(
+                models.F("search_tsvector"), fastupdate=False, name="zerver_message_search_tsvector"
+            ),
         ),
-        migrations.AddField(
-            model_name="realm",
-            name="message_content_edit_limit_seconds",
-            field=models.IntegerField(default=600),
+        migrations.AddIndex(
+            model_name="message",
+            index=models.Index(
+                django.db.models.functions.text.Upper("subject"), name="upper_subject_idx"
+            ),
         ),
-        migrations.AddField(
-            model_name="realm",
-            name="default_language",
-            field=models.CharField(default="en", max_length=50),
+        migrations.AddIndex(
+            model_name="stream",
+            index=models.Index(
+                django.db.models.functions.text.Upper("name"), name="upper_stream_name_idx"
+            ),
         ),
-        migrations.AddField(
-            model_name="userprofile",
-            name="tos_version",
-            field=models.CharField(max_length=10, null=True),
+        migrations.AlterModelManagers(
+            name="userprofile",
+            managers=[
+                ("objects", django.contrib.auth.models.UserManager()),
+            ],
```

Comment:
Pretty sure we can pass `managers=` to the CreateModel: https://github.com/django/django/blob/43cdfa8b20e567a801b7d0a09ec67ddd062d5ea4/django/db/migrations/operations/models.py#L47

I'm not sure why this didn't flatten, given https://github.com/django/django/commit/8f7cda083154a2e97e07115a66ee28be27200348